### ISSUE TITLE
Using ODGI to remove complex regions

### ISSUE THREAD

**Issue by brettChapman** [pangenome/odgi#427]
Hi

I've been following this tutorial to remove complex regions from the pangenome graph: https://odgi.readthedocs.io/en/latest/rst/tutorials/remove_artifacts_and_complex_regions.html

With ```odgi extract``` I have added ```-d``` and ```-e``` as I want to resulting graph to have the same number of paths as the original graph. However I notice the path names renamed with genomic coordinates now. Is there a way to keep the path names the same, with just the sequence name, without having to use ```-M``` with ```odgi viz```?

I use the tutorial the same way with a few changes below:
```
odgi depth -i barley_pangenome_graph_1H.og -w 0:0:1 > barley_pangenome_graph_1H_low_depth.bed	
odgi depth -i barley_pangenome_graph_1H.og -w 100000:500:1000 > barley_pangenome_graph_1H_high_depth.bed

odgi extract -i barley_pangenome_graph_1H.og --threads 10 -c 0 --inverse -d 1000000 -e 100 -b barley_pangenome_graph_1H.regions_to_remove.bed -R barley_pangenome_graph_1H_path_to_fully_retain.txt -o barley_pangenome_graph_1H.clean.og
```

I'll be tweaking ```-d``` and ```-e``` to merge paths, and I modified the window size for the low depth, as I found the graph too fragmented using ```100:0:1``` resulting in seg faults when visualising it. Not all graphs identified high depth regions. Chromosome 1H came back with no high depth regions, whereas 5H, 6H and 7H had a lot of high depth regions. I could tell these were centromere regions as the coordinates were around the mid-region of the chromosome.

Here is the 2D graph with ```-d 1000000 -e 100```:

![barley_pangenome_graph_1H clean_viz_inv_e100](https://user-images.githubusercontent.com/8529807/177084783-e24995d7-a1c3-4515-9462-59e1a921adfc.png)

Once I have a cleaned graph, my intention is to use it to index ```vg index``` and map reads to it with ```vg giraffe```, as any attempt to index with the full graph with included complex regions results in seg faults. This is my attempt to simplify the graph for indexing and read mapping. Thanks.

**Comment by brettChapman** [comment]
With ```-d 10000000 -e 100``` I managed to merge the paths. Still their paths names are now annotated with genomic coordinates. For visual purposes can I display just the sequence name instead of the coordinates too, and would having the path names annotated like this interfere with using the cleaned graph for indexing and read mapping with ```vg giraffe```?

![barley_pangenome_graph_1H clean_viz_inv](https://user-images.githubusercontent.com/8529807/177235849-7d8f1775-0912-4c94-8243-bff597dd995c.png)

**Comment by brettChapman** [comment]
Thanks @ekg that worked.

I've managed to reduce the complexity of the graphs by removing low and high depth regions now. However after merging all the graphs prior to an attempt to distance index the graph, I ran ```vg stats -R```, and my largest snarl is now double what it was before. Should I be setting the window size ```odgi depth -w``` to roughly the snarl size to remove these large snarls? For example the large loop in the above figure I provided. I'm guessing these are from low depth regions, and possibly artefacts?
